3rdparty/jvm/com/chuusai/BUILD

```diff
@@ -1,7 +1,7 @@
 load("@io_bazel_rules_scala//scala:scala.bzl", "scala_library")
```

Comment:
can we change this `load` line since we aren't using `scala_library` anymore? This I think is done in the config yaml.

Reply:
Done